library/ix-dev/community/nginx-proxy-manager/templates/_npm.tpl

```diff
@@ -0,0 +1,103 @@
+{{- define "npm.workload" -}}
+workload:
+  npm:
+    enabled: true
+    primary: true
+    type: Deployment
+    podSpec:
+      hostNetwork: {{ .Values.npmNetwork.hostNetwork }}
+      securityContext:
+        fsGroup: 1000
+      containers:
+        npm:
+          enabled: true
+          primary: true
+          imageSelector: image
+          securityContext:
+            runAsUser: 0
+            runAsGroup: 0
+            readOnlyRootFilesystem: false
+            runAsNonRoot: false
+            capabilities:
+              add:
+                # Needed for: s6-applyuidgid: fatal: unable to setuid: Operation not permitted
+                - SETUID
+                # Needed for: s6-applyuidgid: fatal: unable to set supplementary group list: Operation not permitted
+                - SETGID
+                # Needed for: Used in some s6-overlay startup scripts
+                - CHOWN
+                - DAC_OVERRIDE
+                # Needed for: Nginx Service
+                - FOWNER
+          fixedEnv:
+            PUID: 1000
```

Comment:
Currently it only works for a few PUIDs, there are some upstream bugs lately. Once it's stabilized a bit, we can exposed it to the users.
(For example 568 does not work, but it mostly works for PUIDs 1000+)